backend/src/main/java/com/bakdata/conquery/models/worker/LocalNamespace.java

```diff
@@ -0,0 +1,141 @@
+package com.bakdata.conquery.models.worker;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.function.Function;
+
+import com.bakdata.conquery.io.jackson.Injectable;
+import com.bakdata.conquery.io.jackson.View;
+import com.bakdata.conquery.io.storage.NamespaceStorage;
+import com.bakdata.conquery.models.config.ConqueryConfig;
+import com.bakdata.conquery.models.datasets.Dataset;
+import com.bakdata.conquery.models.datasets.PreviewConfig;
+import com.bakdata.conquery.models.datasets.concepts.select.connector.specific.MappableSingleColumnSelect;
+import com.bakdata.conquery.models.identifiable.CentralRegistry;
+import com.bakdata.conquery.models.identifiable.ids.specific.DatasetId;
+import com.bakdata.conquery.models.index.IndexService;
+import com.bakdata.conquery.models.jobs.JobManager;
+import com.bakdata.conquery.models.jobs.SimpleJob;
+import com.bakdata.conquery.models.query.ExecutionManager;
+import com.bakdata.conquery.models.query.FilterSearch;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.ToString;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+@Getter
+@ToString(onlyExplicitlyIncluded = true)
+@RequiredArgsConstructor
+public class LocalNamespace extends IdResolveContext  implements Namespace {
+
+	private final ObjectMapper preprocessMapper;
```

Comment:
is preprocessMapper ein Name von uns? komisch, dass der hier verwendet wird, wo ja Preprocess schon passiert ist.

Reply:
Ja, existiert so im Namespace: https://github.com/ingef/conquery/blob/b6c19fffca99713c66b67c200b8c881259737296/backend/src/main/java/com/bakdata/conquery/models/worker/Namespace.java#L56
Der wird nur hier benutzt:
https://github.com/ingef/conquery/blob/b6c19fffca99713c66b67c200b8c881259737296/backend/src/main/java/com/bakdata/conquery/models/jobs/ImportJob.java#L86

Reply:
ah, weil es gleich wie der preprocess lesen muss stimmt. Danke